src/aux-server/aux-web/shared/interaction/DragOperation/BaseFileDragOperation.ts

```diff
@@ -369,22 +374,36 @@ export abstract class BaseFileDragOperation implements IOperation {
     }
 
     protected _onDragReleased(calc: FileCalculationContext): void {
+        this._files[0].tags[this._context + '.x'] = this._toCoord.x;
+        this._files[0].tags[this._context + '.y'] = this._toCoord.y;
+
+        const fileTemp = createFile(this._files[0].id, this._files[0].tags);
```

Comment:
Same scenario here as above. Can we rewrite this to not edit `this._files[0].tags`?

Reply:
Will do the same changes as above